draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -395,8 +395,28 @@ discussed in Sections 6, 11.1, and 12 of {{?RFC3649}}.
 
 ### Real-Time Protocols
 
-(TODO: Clarify that real time congestion controls are included, with
-allowances for the poor documentation / open source availability of these)
+General-purpose protocols coexist in the internet with real-time congestion
+control protocols, which in general have finite throughput requirements and
+more strict latency bounds.
+
+{{?RFC8868}} provides suggestions for real-time congestion control design and
+{{?RFC8867}} suggests test cases. {{?RFC9392}} descfibees some considerations
+for the RTP Control Protocol (RTCP). This document does not change the
+informational status of those RFCs.
+
+New proposals SHOULD consider coexistence with widely deployed real-time
+congestion controls. Regrettably, at the time of writing, many algorithms with
```

Comment:
Why not with widely-deploted transports in general here? Why specific to real-time protocols?

Reply:
well, the section is about real-time transports. General purpose protocols are in the section above.

Reply:
Sorry - not my point. I meant: Why not "New real-time congestion control proposals SHOULD consider coexistence with widely deployed real-time and non-real-time congestion controls."

Reply:
Editors meeting: this doc is saying all algs (of any type) must be tested against cubic, etc AND real-time protocols. There are not different standards for different types of proposal.